packages/prisma/extensions/disallow-undefined-delete-update-many.ts

```diff
@@ -0,0 +1,40 @@
+import { Prisma } from "@prisma/client";
+
+export const checkUndefinedInValue = (where: any) => {
+  if (where) {
+    for (const key in where) {
+      // INFO: Since this is for $allModels, we don't have a way to get the correct
+      // where type
+      const whereInput = where[key as any] as any;
+      let message;
+      if (!whereInput) {
```

Comment:
This always fails if a `whereInput` value is false, and you get an error without passing undefined.

Reply:
Nice find, @anikdhabal. And from Prisma's docs, with null, they won't ignore the params, only with undefined so this change is ok.

Reply:
Yes, that's true👍